CHANGELOG-1.1.md

```diff
@@ -0,0 +1,119 @@
+- Support zonal and multi-zonal clusters in deploy/gcp ([#809](https://github.com/pingcap/tidb-operator/pull/809), [@cofyc](https://github.com/cofyc))
+- The ad-hoc backup will fail if use default backup name which is not rendered during helm install in backup-job.yaml. ([#836](https://github.com/pingcap/tidb-operator/pull/836), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Add tidb-lightning support ([#817](https://github.com/pingcap/tidb-operator/pull/817), [@tennix](https://github.com/tennix))
+- Support restore the TiDB cluster from a specified scheduled backup directory ([#804](https://github.com/pingcap/tidb-operator/pull/804), [@onlymellb](https://github.com/onlymellb))
+- Fix tkctl log output exception ([#797](https://github.com/pingcap/tidb-operator/pull/797), [@onlymellb](https://github.com/onlymellb))
+- Add `hostNetwork` field in pd/tikv/tidb spec to make it possible to run tidb components in host network. ([#774](https://github.com/pingcap/tidb-operator/pull/774), [@cofyc](https://github.com/cofyc))
+- Use mdadm and RAID rather than LVM when it is available on GKE ([#789](https://github.com/pingcap/tidb-operator/pull/789), [@gregwebs](https://github.com/gregwebs))
+- Users can now expand cloud storage PV dynamically by increasing PVC storage size. ([#772](https://github.com/pingcap/tidb-operator/pull/772), [@tennix](https://github.com/tennix))
+- Support configuring node image types for PD/TiDB/TiKV node pools. ([#776](https://github.com/pingcap/tidb-operator/pull/776), [@cofyc](https://github.com/cofyc))
+- Add a script to delete unused disk for GKE ([#771](https://github.com/pingcap/tidb-operator/pull/771), [@gregwebs](https://github.com/gregwebs))
+- Support `binlog.pump.config` and `binlog.drainer.config` configurations for pump and drainer ([#693](https://github.com/pingcap/tidb-operator/pull/693), [@weekface](https://github.com/weekface))
+- Prevent pump progress from an exit with 0 if the pump become `offline` ([#769](https://github.com/pingcap/tidb-operator/pull/769), [@weekface](https://github.com/weekface))
+- Introduce a new helm chart,  tidb-drainer, to facilitate the multiple drainers management. ([#744](https://github.com/pingcap/tidb-operator/pull/744), [@aylei](https://github.com/aylei))
+- Add backup-manager tool for support backup、restore and clean backup data ([#694](https://github.com/pingcap/tidb-operator/pull/694), [@onlymellb](https://github.com/onlymellb))
+- Add `affinity` to pump/drainer configration ([#741](https://github.com/pingcap/tidb-operator/pull/741), [@weekface](https://github.com/weekface))
+- Fix tikv scale in failure in some cases after tikv failover ([#726](https://github.com/pingcap/tidb-operator/pull/726), [@onlymellb](https://github.com/onlymellb))
+- `NONE` ([#719](https://github.com/pingcap/tidb-operator/pull/719), [@xiaojingchen](https://github.com/xiaojingchen))
```

Comment:
?

Reply:
There is no user-facing change in this PR, I will remove this